Metalama.Framework.Engine/Utilities/Roslyn/SerializableDeclarationIdProvider.cs

```diff
@@ -41,6 +41,7 @@ private static bool TryGetSerializableId( this ISymbol? symbol, DeclarationRefTa
         {
             case null:
             case ILocalSymbol:
+            case { Kind: SymbolKind.Alias }:
```

Comment:
I think this is a wrong approach. We should serialize the target symbol of the alias.

Reply:
I think since this is SerializableDeclarationId, the alias should not be followed because we want the Id of the alias itself. The target can be non-declaration itself (e.g. `List<int>` or `int*`). I'll return the failure back in and handle it in refactoring provider, which should ignore aliases.